pkg/controller/pravega/pravega_controller_test.go

```diff
@@ -103,8 +103,9 @@ var _ = Describe("Controller", func() {
 						},
 					},
 					Authentication: &v1beta1.AuthenticationParameters{
-						Enabled:            true,
-						PasswordAuthSecret: "authentication-secret",
+						Enabled:               true,
+						PasswordAuthSecret:    "authentication-secret",
+						ControllerTokenSecret: "controllerauthsecret",
```

Comment:
For consistency, maybe controllerauthsecret --> controllerauth-secret ?

Reply:
I have changed it.